cluster-scope/base/namespaces/openshift-monitoring/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: openshift-monitoring
+
+resources:
+  - metrics-clusterrole.yaml
+  - metrics-clusterrolebinding.yaml
```

Comment:
I've also noticed, you're defining a `cluster-scope/base/namespaces/<NAMESPACE_NAME>` without any `Namespace` kind resource. Why is that? IIf you are defining a namespace, a namespace resource should be present

Reply:
Ignoring the fact that these resources should not be under a namespace (in agreement with your other comment), 

AFAIK this namespace gets created upon openshift install, so it's expected to be there. We probably should be careful over-writing these namespaces, a quick check shows that this particular namespace has specific labels and annotations that look to have been auto-generated.

Reply:
so this is because this namespace is automatically created by openshift and I thought we should not mess with an `openshift-` namespace